src/CoreWCF.Primitives/src/CoreWCF/Channels/MessageFault.cs

```diff
@@ -104,6 +107,19 @@ public static MessageFault CreateFault(Message message, int maxBufferSize)
             }
         }
 
+        public static MessageFault Default
+        {
```

Comment:
MessageFault.Default is only used by WebHttp. Move into WebHttp either into the consuming class or as a separate static helper class. You will need to use a difference overload of CreateFault which is public, but if you follow the call path through, this call ends up at the public method via multiple other methods which populate default values for the other arguments.,